File: testmemorylink/src/main/java/com/example/testmemorylink/MainActivity.java
package com.example.testmemorylink;

import android.app.Activity;
import android.content.Intent;
import android.os.Bundle;
import android.view.View;

public class MainActivity extends Activity implements View.OnClickListener{


    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);


    }

    @Override
    public void onClick(View v) {
        Intent intent = new Intent(this,SecondActivity.class);
        startActivity(intent);
    }
}

File: myviewmodel/src/main/java/com/example/myviewmodel/MainActivity.kt
package com.example.myviewmodel

import android.app.Activity
import android.arch.lifecycle.MutableLiveData
import android.arch.lifecycle.ViewModel
import android.content.Intent
import android.os.Bundle

class MainActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        var intent = Intent(this,SecondActivity::class.java);
        startActivity(intent)

    }




}

//class MyViewModel():ViewModel(){
//    lateinit var users: MutableLiveData<List<User>>;
//    fun getUsers()->{
//
//    }
//}

data class User(var name:String,var age:String)
